pkg/contracts/init.go

```diff
@@ -0,0 +1,84 @@
+package contracts
+
+import (
+	"context"
+	"os"
+
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+
+	"github.com/CoreumFoundation/coreum-tools/pkg/libexec"
+	"github.com/CoreumFoundation/coreum-tools/pkg/logger"
+)
+
+// InitConfig provides params for the init stage.
+type InitConfig struct {
+	// TargetDir is OS path to the target dir where contract source will be staged. It will be created if not existing.
+	TargetDir string
+	// TemplateRepoURL is public Git repo URL to clone smart-contract template from
+	TemplateRepoURL string
+	// TemplateVersion specifies the version of the template, e.g. 1.0, 1.0-minimal, 0.16
+	TemplateVersion string
+	// TemplateSubdir specifies a subfolder within the template repository to be used as the actual template
+	TemplateSubdir string
+	// ProjectName is the smart-contract name used in template scaffolding
+	ProjectName string
+}
+
+func (c InitConfig) Sanitize() InitConfig {
```

Comment:
As I see, all exported types have comments. Shouldn't this function be commented on as well?